WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java

```diff
@@ -194,7 +193,7 @@ public void onErrorResponse(VolleyError response) {
                             }
                             AppLog.d(T.THEMES, getString(R.string.theme_auth_error_authenticate));
                         } else {
-                            Toast.makeText(ThemeBrowserActivity.this, R.string.theme_fetch_failed, Toast.LENGTH_LONG)
+                            ToastUtils.showToast(ThemeBrowserActivity.this, R.string.theme_fetch_failed, ToastUtils.Duration.LONG)
                                     .show();
```

Comment:
Even after reviewing the PR twice, I still missed one 👎 Fixed in 8b3acc86f7d7d58fae640229e3f2024672e40fb7